src/api/model/user_model

```diff
@@ -0,0 +1,12 @@
+from pydantic import BaseModel
+from datetime import date
+
+class User(BaseModel):
+    id: int
+    name: str
+    age: int
```

Comment:
Age can be calculated from date of birth, so age field can be removed.

Reply:
Setting age field in User is a good approach. Update on creation or editing the dob field.
Better to set at: Event listeners / Object Relational Model
Why: Instead of calculating at everytime.
Faster